code/pizzabot.js

```diff
@@ -4,4 +4,88 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+var x = vegetarian
+var y = hawaiian
+var z = pepperoni
+
+alert("Hey! Happy to serve you pizza! On our menu we have " + x + ", " + y + " and " + z + ".");
+
+
+let orderName = prompt("Enter the name of the pizza you want to order today.");
+
+
+if (orderName.toLowerCase() == "vegetarian") {
+  modifiedName = x;
+} else if (orderName.toLowerCase() == "vegetarian pizza") {
+  modifiedName = x;
+} else if (orderName.toLowerCase() == "hawaiian") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "hawaiian pizza") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "hawaian") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "hawaian pizza") {
+  modifiedName = y;
+} else if (orderName.toLowerCase() == "pepperoni") {
+  modifiedName = z;
+} else if (orderName.toLowerCase() == "pepperoni pizza") {
+  modifiedName = z;
+} else if (orderName.toLowerCase() == "peperoni") {
+  modifiedName = z;
+} else if (orderName.toLowerCase() == "peperoni pizza") {
+  modifiedName = z;
+} else {
+  alert('Select a pizza from the menu');
+  let orderName = prompt("Enter the name of the pizza you want to order today.");
+  if (orderName.toLowerCase() == "vegetarian") {
+    modifiedName = x;
+  } else if (orderName.toLowerCase() == "vegetarian pizza") {
+    modifiedName = x;
+  } else if (orderName.toLowerCase() == "hawaiian") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "hawaiian pizza") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "hawaian") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "hawaian pizza") {
+    modifiedName = y;
+  } else if (orderName.toLowerCase() == "pepperoni") {
+    modifiedName = z;
+  } else if (orderName.toLowerCase() == "pepperoni pizza") {
+    modifiedName = z;
+  } else if (orderName.toLowerCase() == "peperoni") {
+    modifiedName = z;
+  } else if (orderName.toLowerCase() == "peperoni pizza") {
+    modifiedName = z;
+  } else {
+    alert('Select a pizza from the menu');
+  }
+}
+
+
+const orderQuantity = prompt("How many of " + modifiedName + " do you want?");
+
+const totalCost = orderQuantity * 80;
+
+if (orderQuantity < 3) {
+  cookingTime = "10";
+} else if (orderQuantity < 6) {
+  cookingTime = "15";
+} else {
+  cookingTime = "20";
+} 
+
+console.log(vegetarian, hawaiian, pepperoni, pizzaPrice)
+console.log(orderName);
+console.log(orderQuantity);
+console.log(`The customer has ordered ${orderQuantity} of ${orderName}.`);
+console.log('It will cost ' + orderQuantity * 80 + 'kr.');
+console.log('It will take ' + cookingTime + ' to prepare.')
```

Comment:
Using `console.log()` is meant to be for developing purpose (so you can check so everything works) but you can delete it when you are making a pull request!